Entities/BlockChangeRequest.cs

```diff
@@ -7,14 +7,14 @@ namespace FRITeam.Swapify.Entities
 {
     public class BlockChangeRequest : BaseEntity
     {
-        public List<Block> BlockOpitonsChange { get; set; }
-        public Block BlockForChange { get; set; }
-        public Student Student { get; set; }
-        public eStatusType Status { get; set; }
+        public List<Guid> BlockOptionsChange { get; set; }
```

Comment:
I'm not sure if this blocks will exists in database (so no Id), may be this will be composition.. But let it be for now.. Probably we will have to redesign whole structure